maps/models.py

```diff
@@ -457,6 +457,20 @@ class Map(models.Model):
         null=True, blank=True,
     )
 
+    # Vulnerability block
+    has_vulnerable_population = models.BooleanField(
+        default=False,
+        help_text="Does the map show information on specific vulnerabilities of the population." 
+    )
+    vulnerable_population_data_date = models.DateField(
```

Comment:
In other sections I added earliest and latest because the field has to be able to cope with ranges as well as single dates. I'm not sure this is the best strategy but having that gives us options to convert it out later if needs be